### PR TITLE
Fluxbox switch source

### PR DESCRIPTION
**Pull request by zlice**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture, x86_64

#### Comments

Ran into issues with the sourceforge tar using pregenerated aclocal.m4/make files. Saw that github is getting commits here and there and the source has autogen.sh so I built from there.

There are a few file differences from the sourceforge tar. From what I can tell:

 - a `defaults.cc` generated by a makefile?
 - some `tests/` are in github that aren't in sourceforge
 - already made languages files in sourceforge
 - updated docs that aren't in sourceforge

### CODE REVIEW

File: srcpkgs/fluxbox/template [hunk 1]
```diff
@@ -10,17 +11,21 @@ configure_args="--enable-nls
  ac_cv_lib_Xrandr_XRRQueryExtension=yes ac_cv_lib_fribidi_fribidi_version_info=yes"
 hostmakedepends="pkg-config"
 makedepends="libSM-devel libXrender-devel libXft-devel libXpm-devel imlib2-devel
- libXext-devel libXinerama-devel libXrandr-devel fribidi-devel"
+ libXext-devel libXinerama-devel libXrandr-devel fribidi-devel automake libtool
+ gettext gettext-devel"
```
Comment (classabbyamp) [inline srcpkgs/fluxbox/template:15]: automake, libtool, and gettext (and maybe gettext-devel) should probably be hostmakedeps

File: srcpkgs/fluxbox/template [hunk 2]
```diff
@@ -10,17 +11,21 @@ configure_args="--enable-nls
...
+homepage="http://www.fluxbox.org"
+distfiles="https://github.com/fluxbox/fluxbox/archive/refs/tags/Release-${version//./_}.tar.gz"
+checksum=990414ae342a35c5703e5025a8bf019b0b4dada55d57071c51d3d498abb5cd36
 
 case "$XBPS_TARGET_MACHINE" in
 	*-musl) configure_args="${configure_args/enable-nls/disable-nls}" ;;
 esac
+pre_configure() {
+	./autogen.sh
+}
```
Comment (classabbyamp) [inline srcpkgs/fluxbox/template:28]: nitpick: add an empty line between the esac, this function, and the next function

File: srcpkgs/fluxbox/template (hunk 1 above)
Comment (zlice) [reply srcpkgs/fluxbox/template]: read but dont understand the diff. 1 for build and the other for some flag that installs sources for you or something?

**Comment by classabbyamp** [comment]
please rebase on master

**Pull request merged**